src/main/java/seedu/address/ui/SlotCard.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.slot.Slot;
+
+/**
+ * An UI component that displays information of a {@code Slot}.
+ */
+public class SlotCard extends UiPart<Region> {
```

Comment:
Im concerned with how we gonna do the inheritance later for alternative displaymethods (we wanted to do timetable also?)

Reply:
was thinking since we currently use a `DisplayListPanel`, we could have something like `DisplayTablePanel` for a tabular view but haven't really given it much thought beyond that